prql-compiler/src/tests/test_bad_error_messages.rs

```diff
@@ -71,6 +71,20 @@ fn test_bad_error_messages() {
        │     Note: Type `relation` expands to `[tuple_of_scalars]`
     ───╯
     "###);
+
+    // It's better if we can tell them to put in [] bracketß
```

Comment:
```suggestion
    // It's better if we can tell them to put in {} braces
```

Reply:
Is the book outdated too? Cuz I see square bracket being used there (screenshot below)
![Screenshot 2023-06-20 at 1 16 44 AM](https://github.com/PRQL/prql/assets/46258781/b6289ae8-70a6-46c9-ba88-1ad5cd96c71b)

Reply:
Yes, all the docs are on 0.8.1; then `main` has the new syntax

Reply:
`task run-book` to see the current book locally...